examples/favorites_advanced/rx_bloc_favorites_advanced/lib/feature_home/views/home_page.dart

```diff
@@ -30,7 +30,7 @@ class HomePage extends StatelessWidget with AutoRouteWrapper {
 
   @override
   Widget build(BuildContext context) => Scaffold(
-        resizeToAvoidBottomPadding: false,
+        // resizeToAvoidBottomPadding: false,
```

Comment:
@RomanovaPrime, do we really need to comment this line ?

Reply:
Replaced resizeToAvoidBottomPadding with resizeToAvoidBottomInset